SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
+reported vulnerability, what to expect if the vulnerability is accepted or
+declined, etc.
+ WHERE TO REPORT SECURITY VULNERABILITIES
```

Comment:
As @qqmyers already mentioned, security@dataverse.org is the place to report security vulnerabilities. This is mentioned under "Reporting Issues and Contributing" at https://guides.dataverse.org/en/5.8/ as well as CONTRIBUTING.md, the issue template, and https://dataverse.org/contact

This issue is related: "Securing Your Installation" section of Installation Guide could cover ongoing security, advisories, private discussion #3215